ERCS/erc-draft-forkable-token.md

```diff
@@ -0,0 +1,262 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8054
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.